.github/workflows/main.yml

```diff
@@ -222,7 +222,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ['3.9']
+        python-version: ['3.11']
```

Comment:
Doing a separate PR to update all of these now that 3.8 is basically at EOL

Reply:
Cool! I've removed this commit